worker/src/RTC/PlainTransport.cpp

```diff
@@ -104,13 +104,14 @@ namespace RTC
 
 		if (options->enableSrtp())
 		{
-			if (!options->srtpCryptoSuite().has_value())
+			auto srtpCryptoSuite = options->srtpCryptoSuite();
+			if (!srtpCryptoSuite.has_value())
```

Comment:
`options->srtpCryptoSuite()` returns an optional, meaning previous code:

```cpp
if (!options->srtpCryptoSuite().has_value())
{ 
  this->srtpCryptoSuite = SrtpSession::CryptoSuiteFromFbs(options->srtpCryptoSuite().value());
```

Here the call to `options->srtpCryptoSuite()` in the conditional statement and the call to it in the conditional body return **different** objects, hence there is no way to static analize the second as being previously checked before accessing.

The solution is to store the result of `options->srtpCryptoSuite()` into a variable and then the condition can be statically analysed and assure that we are accessing a checked option value, since the object in the condition statement and body are the same.